Paypal Integration Guide (#59)
paypal/iOS-SDK: docs/PayPalWebCheckout/README.md

# Accepting PayPal Web Checkout Payments

> The PayPal Web Checkout module in the PayPal SDK enables PayPal payments in your app.

Reviewer: ```suggestion
The PayPal Web Checkout module in the PayPal SDK enables PayPal payments via a webview in your app.
```

Follow these steps to add PayPal Web Checkout payments:

1. [Setup a PayPal Developer Account](#setup-a-paypal-developer-account)
1. [Add PayPal Web Checkout Module](#add-paypal-web-checkout-module)
1. [Test and go live](#test-and-go-live)

## Setup a PayPal Developer Account

You will need to set up authorization to use the PayPal Payments SDK. 
Follow the steps in [Get Started](https://developer.paypal.com/api/rest/#link-getstarted) to create a client ID. 

You will need a server integration to create an order to capture funds using the [PayPal Orders v2 API](https://developer.paypal.com/docs/api/orders/v2). 
For initial setup, the `curl` commands below can be used as a reference for making server-side RESTful API calls.

## Add PayPal Web Checkout Module

### 1. Add the Payments SDK  to your app

#### Swift Package Manager

In Xcode, add the PayPal SDK as a [package dependency](https://developer.apple.com/documentation/swift_packages/adding_package_dependencies_to_your_app) to your Xcode project. Enter https://github.com/paypal/iOS-SDK as the package URL. Tick the "PayPalWebCheckout" checkbox to add the PayPal Web Checkout package to your app.

> In your app's source code files, use the following import syntax to include the PayPal Card module:

Reviewer: ```suggestion
In your app's source code files, use the following import syntax to include the PayPal Web Checkout module:
```

let payPalRequest = PayPalWebCheckoutRequest(orderID: "<ORDER_ID>")
```

> You can also specify one of the follwing funding sources for your order: `PayPal (default)`, `PayLater` or `PayPalCredit`.

Reviewer: ```suggestion
You can also specify one of the following funding sources for your order: `PayPal` (default), `PayLater` or `PayPalCredit`.
```
